packages/react-native-web/src/exports/Picker/index.js

```diff
@@ -12,17 +12,19 @@
 
 import applyNativeMethods from '../../modules/applyNativeMethods';
 import { Component } from 'react';
-import ColorPropType from '../../propTypes/ColorPropType';
-import createElement from '../../modules/createElement';
+import ColorPropType from '../ColorPropType';
+import createElement from '../createElement';
 import PickerItemPropType from './PickerItemPropType';
-import PickerStylePropTypes from './PickerStylePropTypes';
-import StyleSheetPropType from '../../propTypes/StyleSheetPropType';
-import StyleSheet from '../../apis/StyleSheet';
-import TextStylePropTypes from '../Text/TextStylePropTypes';
+import StyleSheetPropType from '../../modules/StyleSheetPropType';
+import StyleSheet from '../StyleSheet';
+import TextPropTypes from '../Text/TextPropTypes';
+import ViewPropTypes from '../ViewPropTypes';
 import { arrayOf, bool, func, number, oneOfType, string } from 'prop-types';
 
-const pickerStyleType = StyleSheetPropType(PickerStylePropTypes);
-const itemStylePropType = StyleSheetPropType(TextStylePropTypes);
+const pickerStyleType = StyleSheetPropType({
+  ...ViewPropTypes.style,
```

Comment:
Looks like a bug, missing import of `PickerStylePropTypes` but should be using that